_data/projects/Sked.yml

```diff
@@ -0,0 +1,11 @@
+name: Sked
+desc: Container scheduling and management, focused on operational simplicity.
+site: https://github.com/ColDog/sked
+tags:
+  - Go
+  - schedulers
+  - Consul
+
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/ColDog/sked
```

Comment:
This link should refer to the list of issues available for contributors, rather than the repository home page.